src/mc-fle2-payload-iev-private.h

```diff
@@ -103,11 +103,34 @@ mc_FLE2IndexedEncryptedValue_new (void);
 mc_FLE2IndexedEqualityEncryptedValueTokens *
 mc_FLE2IndexedEqualityEncryptedValueTokens_new (void);
 
+/**
+ * This function is used by the server codebase.
+ */
+bool
+mc_FLE2IndexedEqualityEncryptedValueTokens_init_from_buf (
```

Comment:
```suggestion
mc_FLE2IndexedEqualityEncryptedValueTokens_init_from_buffer (
```
Suggest renaming for consistency with other `*_from_buffer` functions, i.e. `mc_reader_init_from_buffer`.